File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/02.Conditional-Statements/02-exercise/06.worldSwimmingRecord.js
function worldSwimmingRecord(input) {

    const recordInSec = Number(input[0]);
    const distanceInMeters = Number(input[1]);
    const timeInSecPerMeter = Number(input[2]);

    const lagInSec = Math.floor(distanceInMeters / 15) * 12.5;
    const ivanTime = distanceInMeters * timeInSecPerMeter + lagInSec;
    const difference = Math.abs(recordInSec - ivanTime);

    if (ivanTime < recordInSec) {
        console.log(`Yes, he succeeded! The new world record is ${ivanTime.toFixed(2)} seconds.`);
    } else {
        console.log(`No, he failed! He was ${difference.toFixed(2)} seconds slower.`);
    }

}
worldSwimmingRecord(["55555.67", "3017", "5.03"]);

File: Python_Basics/Homeworks-And-Labs/02.conditional_statements/02.exercise/06.world_swimming_record.py
recordInSec = float(input())
distanceInMeters = float(input())
timeInSecPerMeter = float(input())

lagInSec = (distanceInMeters // 15) * 12.5
ivanTime = distanceInMeters * timeInSecPerMeter + lagInSec
difference = abs(recordInSec - ivanTime)

if ivanTime < recordInSec:
    print(f'Yes, he succeeded! The new world record is {ivanTime:.2f} seconds.')
else:
    print(f'No, he failed! He was {difference:.2f} seconds slower.')